test/unit/test_h5_loader.py

```diff
@@ -63,6 +64,22 @@ def test_set_group_structure():
     assert got == expected
 
 
+def test_set_group_structure_ungrouped():
+    """
+    check if the set_group_structure method works as intended when data is
+    not grouped
+    """
+    dir_path = "data/test/h5/paired/test"
```

Comment:
Change to ./data/test...? relative file path works locally for me vs the current solution. Will it pass travis?